### PR TITLE
Added `chat_update_message` method

### PR DESCRIPTION
**Pull request by hery**

### CODE REVIEW

File: pyslack/__init__.py [hunk 1]
```diff
@@ -41,6 +41,22 @@ def chat_post_message(self, channel, text, **params):
         })
         return self._make_request(method, params)
 
+    def chat_update_message(self, channel, text, timestamp, **params):
+        """chat.update
+
+        This method updates a message.
```
Comment (loisaidasam) [inline pyslack/__init__.py]: So after reading the docs, I see that `ts` has to be the timestamp of the original message sent - can you make a comment here specifying that?

Comment (hery) [inline pyslack/__init__.py]: Yes!

Comment (hery) [inline pyslack/__init__.py]: But they say "message to be updated," so if we're updating an updated message, it should be the timestamp of the updated message, right?

Comment (hery) [inline pyslack/__init__.py]: Hmm, forgot to escape the ` 🙊

File: pyslack/__init__.py [hunk 2]
```diff
@@ -41,6 +41,27 @@ def chat_post_message(self, channel, text, **params):
...
+
+        Check docs for all available **params options:
+        https://api.slack.com/methods/chat.update
+
+        Required parameters:
+        `ts`:  Timestamp of the message to be updated (e.g: 1405894322.002768)
```
Comment (loisaidasam) [inline pyslack/__init__.py]: can we add quotes around these `e.g:` params to signify that they're all strings?

Comment (hery) [inline pyslack/__init__.py]: Totes!

**Comment by loisaidasam** [comment]
Hell yeah! :+1:

**Comment by hery** [comment]
Thank you 🙇

**Pull request merged**